app/src/main/java/com/android/consumerfinancehalan/utils/ApiResponse.kt

```diff
@@ -0,0 +1,8 @@
+package com.android.consumerfinancehalan.utils
```

Comment:
this file jas nothing to do with the api response , 
for example , loading state has nothing to do with the response or the api call made